main/guides/js-programming/agoric-js-overview.md

```diff
@@ -107,13 +107,10 @@ additions and not part of JavaScript.
 Most Node.js-specific [global objects](https://nodejs.org/dist/latest-v14.x/docs/api/globals.html) 
 are unavailable within a vat including:
 
-* `queueMicrotask`
+* `queueMicrotask`: You can 
+  replace `queueMicrotask(fn)` with `Promise.resolve().then(_ => fn())`.
```

Comment:
Are you sure that's true? Does `queueMicrotask` use the promise queue, or (as I'd expect) does it use another queue with a lower priority, like `setImmediate`, etc?

Reply:
100% sure: https://twitter.com/mhofman/status/1327729938640773120